internal/batches/ui/exec_ui.go

```diff
@@ -22,11 +22,8 @@ type ExecUI interface {
 	DeterminingWorkspaceCreatorType()
 	DeterminingWorkspaceCreatorTypeSuccess(wt workspace.CreatorType)
 
-	ResolvingRepositories()
-	ResolvingRepositoriesDone(repos []*graphql.Repository, unsupported batches.UnsupportedRepoSet, ignored batches.IgnoredRepoSet)
-
 	DeterminingWorkspaces()
-	DeterminingWorkspacesSuccess(num int)
+	DeterminingWorkspacesSuccess(workspacesCount, reposCount int, unsupported batches.UnsupportedRepoSet, ignored batches.IgnoredRepoSet)
```

Comment:
Nit, but it looks like for `unsupported` and `ignored` we actually also only use them to get the counts with `len`. Could we just pass all 4 as counts?

Reply:
The TUI prints them, so we need to keep it for this one